models/namespace.js

```diff
@@ -74,8 +79,16 @@ export default {
     return false;
   },
 
+  // These are namespaces that are created by rancher to serve purposes in the background but the user shouldn't have
+  // to worry themselves about them.
+  isObscure() {
+    return OBSCURE_NAMESPACE_PREFIX.some(prefix => this.metadata.name.startsWith(prefix)) && this.isSystem;
```

Comment:
Would it be possible to gate this on the `dev` preference (`this.$store.getters['prefs/get'](DEV)`)? I could see it being handy to manage these in some dev cases

Reply:
I think this may end up causing confusion down the line where results don't line up but let's go ahead and give it a shot.

I performed the check in projectsnamespaces as it seems to fit better there.